modules/openapi-generator/src/main/resources/java-micronaut/common/model/pojo.mustache

```diff
@@ -404,4 +404,4 @@ Declare the class with extends and implements
         {{/interfaces.0}}
     {{/parent}}
     {{/visitable}}
-}
\ No newline at end of file
+}
```

Comment:
can you remove new line here? will reduce number of updates in samples

Reply:
sure